test/integration/create-next-app/index.test.js

```diff
@@ -229,43 +237,29 @@ describe('create next app', () => {
 
   it('should exit if example flag is empty', async () => {
     await usingTempDir(async (cwd) => {
-      try {
-        const projectName = 'no-example-provided'
-        await run(cwd, [projectName, '--example'])
-      } catch (e) {
-        // eslint-disable-next-line jest/no-try-expect
-        expect(e.exitCode).toBe(1)
-      }
+      const projectName = 'no-example-provided'
+      const res = await run([projectName, '--example'], { cwd, reject: false })
+      expect(res.exitCode).toBe(1)
     })
   })
 
-  it('should exit if the folder is not writable', async () => {
-    await usingTempDir(async (cwd) => {
-      const projectName = 'not-writable'
-      expect.assertions(2)
-      try {
-        const res = await run(cwd, [projectName])
-
-        if (process.platform === 'win32') {
-          expect(res.exitCode).toBe(0)
-          expect(
-            fs.existsSync(path.join(cwd, projectName, 'package.json'))
-          ).toBeTruthy()
-        }
-      } catch (e) {
-        // eslint-disable-next-line jest/no-try-expect
-        expect(e.exitCode).toBe(1)
-        // eslint-disable-next-line jest/no-try-expect
-        expect(e.stderr).toMatch(
+  if (process.platform !== 'win32') {
+    it('should exit if the folder is not writable', async () => {
+      await usingTempDir(async (cwd) => {
+        const projectName = 'not-writable'
+        const res = await run([projectName], { cwd, reject: false })
+
+        expect(res.exitCode).toBe(1)
+        expect(res.stderr).toMatch(
           /you do not have write permissions for this folder/
         )
-      }
-    }, 0o500)
-  })
+      }, 0o500)
+    })
+  }
```

Comment:
Seems you stopped testing windows all together here. Could you please re-introduce it?